devel/nodejs15/Portfile

```diff
@@ -69,6 +56,36 @@ proc rec_glob {basedir pattern} {
     return $files
 }
 
+if { ${os.platform} eq "darwin" && ${os.major} < 11 } {
+    # on macOS nodejs only builds against libc++
+    # this force is OK as node does not link against any other c++ libs
+    # per @kencu, libcxx is only for 10.5 and 10.6 as a prerequisite for an even-newer libcxx (macports-libcxx)
+    depends_lib-append      port:libcxx
```

Comment:
Did MacPorts switch 10.5 to `libc++`? If not, this is wrong. Instead, it should use gcc to build there.